<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pacsPage</class>
 <widget class="QWidget" name="pacsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PACS</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblPacsType">
     <property name="text">
      <string>&amp;Тип</string>
     </property>
     <property name="buddy">
      <cstring>cmbPacsType</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cmbPacsType">
     <item>
      <property name="text">
       <string>Выполнить команду</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Weasis</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Открыть браузер</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>164</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblPacsParams">
     <property name="text">
      <string>&amp;Параметры</string>
     </property>
     <property name="buddy">
      <cstring>edtPacsParams</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="edtPacsParams"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblPacsWado">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Wado</string>
     </property>
     <property name="buddy">
      <cstring>edtPacsWado</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="edtPacsWado">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>214</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbPacsType</tabstop>
  <tabstop>edtPacsParams</tabstop>
  <tabstop>edtPacsWado</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
